client/src/components/PageTemplate/PageTemplate.scss

```diff
@@ -13,7 +13,7 @@
     @media (min-width: 1500px) {
       display: flex;
       flex-direction: column;
-      width: 1300px;
+      width: 1380px;
```

Comment:
Did you make sure that this works alright on all screen sizes and pages?

Reply:
in all tests I conducted it worked fine